config.yml.dist

```diff
@@ -1,6 +1,9 @@
 default:
-  rooturl: http://localhost:8088/rest
+  rooturl: http://localhost:8080/rest
   admin-user: fedoraAdmin
   admin-password: fedoraAdmin
   user: testuser
   password: testpass
+  broker-url: tcp://127.0.0.1:61616
```

Comment:
@whikloj : you also need to add the broker-url,topic, and/or queue name to the command line parameters since the config file is optional.  Without -c The application hangs.

Reply:
Do you want new CLI options for these 3, or just have it verify their existence in the config file?

Reply:
I would say for consistency add them to CLI options and make the tool fail if they are not configured in either place.